src/api/test/unit/code_quality_test.rb

```diff
@@ -77,7 +77,7 @@ def setup
       'BsRequestActionSubmit#execute_accept'                                    => 126.42,
       'BsRequestPermissionCheck#cmd_changestate_permissions'                    => 117.09,
       'RequestSourceDiff::ActionSourceDiffer#diff_for_source'                   => 94.62,
-      'BuildController#file'                                                    => 127.42,
+      'BuildController#file'                                                    => 130.44,
```

Comment:
because it's an ongoing effort - and it's not as easy as running rubocop auto-corrects. It's real work, but brings real benefits.